CHANGELOG_iOS.md

```diff
@@ -1,5 +1,51 @@
 # Changelog
 
+## [1.75.179](https://github.com/brave/brave-browser/releases/tag/v1.75.179)
+
+### Web3
+
+ - Added new updates to the transaction status screen. ([#26146](https://github.com/brave/brave-core/pull/26146))
+ - Added “Speed up” button on wallet transaction status screen. ([#41800](https://github.com/brave/brave-browser/issues/41800))
+ - Added “Cancel Transaction” button on wallet transaction status screen. ([#41799](https://github.com/brave/brave-browser/issues/41799))
+ - Added support for resolving new Unstoppable Domains TLDs. ([#42368](https://github.com/brave/brave-browser/issues/42368))
+
+### General
+
+ - Added new customizable menu UI behind the “#brave-use-modern-browser-menu” flag. ([#42836](https://github.com/brave/brave-browser/issues/42836))
+ - Added Smart-Proxy and Kill-Switch support for Brave VPN. ([#42275](https://github.com/brave/brave-browser/issues/42275))
+ - Added Shred feature to the contextual menu. ([#42240](https://github.com/brave/brave-browser/issues/42240))
+ - Added support for custom site distillers. ([#40794](https://github.com/brave/brave-browser/issues/40794))
+ - Added brave://ads-internals/ page for managing ads data. ([#40952](https://github.com/brave/brave-browser/issues/40952))
```

Comment:
I don't think we mentioned `brave://ads-internals` in the `Desktop` or `Android` release notes. But I'm fine leaving it in. Just figured I would mention it. If we're keeping it, lets remove the `/` so it's `brave://ads-internals`. Usually we remove the trailing `/`.